ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/objectstore/StoreManager.java

```diff
@@ -145,20 +145,20 @@ private static final ObjectStoreAPI initStore(String name)
     {
         ObjectStoreEnvironmentBean storeEnvBean = BeanPopulator.getNamedInstance(ObjectStoreEnvironmentBean.class, name);
         String storeType = storeEnvBean.getObjectStoreType();
-        ObjectStoreAPI store;
 
         try
         {
-            store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+            ObjectStoreAPI store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+
+            store.start();
```

Comment:
As unlikely as it might be, people might be expecting the NPE now so it could constitute a public API change which is not a good compromise if its just for aesthetics.